packages/appcd-gulp/src/babel.js

```diff
@@ -1,4 +1,8 @@
-{
+const fs     = require('fs');
```

Comment:
The babel changes here broke running tests. I have changes locally that make this export the profiles as profiles and the function it does currently as `init`, so the changes you made would be `require('./babel.js').init(opts)` and the test require becomes `const conf = require('./babel.js').profiles[process.env.APPCD_BABEL_CONF || 'node8'];`, shall I push or do you wish to fix yourself?

Reply:
Fixed.